01-cloudformation/README.md

```diff
@@ -318,6 +337,8 @@ functionality. Query S3 to ensure that the buckets have been deleted.
 _Can you list 4 features of CloudFormation that help make a CFN template
 portable code?_
 
+#### The usage of parameters, mappings, conditions, and imports
```

Comment:
Portability is a measure of how easily an application can be transferred from one computer environment to another. In other words, portable code is code that is not tightly coupled to one specific platform. CloudFormation templates show portability in the following ways:
1. CFN templates can be written in YAML or JSON, which are predefined markup languages that are not dependent on systems but their own compilers that run on different systems. This means using a YAML or JSON CFN template across different systems will be semantically the same.
2. CFN templates can pull account-based and resource-based property data predefined by AWS CloudFormation by use of pseudo-parameters and intrinsic functionalities (i.e. Mappings, Conditions, !Ref, !GetAtt) instead of hard-coding them, all of which work the same across all systems that support YAML or JSON compilers.
3. CloudFormation has a web designer for CFN templates, making the functionality of template creation even more portable as the technology backing such a software tool is managed and delivered by them over one architecture. You can even create CFN templates from any web-enabled device essentially!
4. Finally, and perhaps one of the most important facets of CloudFormation portability, is that the same single CFN template is able to be instigated across multiple if not all AWS regions. Additionally, CFN templates can depend on resource output values of other CFN templates in the same region, decreasing need to repeat functional 'gets' of values attributed to resources.